cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -184,8 +183,8 @@ public void setVoltageAngleNodeBreaker() {
                 return;
             }
             LOG.warn(
-                "Can't find a bus from the Bus View to set Voltage and Angle, we use the bus {} from the Bus/Breaker view. Connectivity node {}",
-                bus, id);
+                    "Can't find a calculated Bus to set Voltage, Angle, but found a configured Bus {}. Connectivity node {}",
+                    bus, id);
```

Comment:
Same question?

Reply:
@lcosson: any feedback

Reply:
Very sorry about that, it looks like I miss-merged this file. Thank God for PR's!
Correcting...